### ISSUE TITLE
When Date stated in string format floating use starting characters only

### ISSUE THREAD

**Issue by tgely** [shuchkin/simplexlsx#123]
Hi @shuchkin ,

I like this code!

https://github.com/shuchkin/simplexlsx/blob/0de04637f322df8a0ed233339d182c77fb4d0e9c/src/SimpleXLSX.php#L975

Example:
```XML
                                    [3] => SimpleXMLElement Object
                                        (
                                            [@attributes] => Array
                                                (
                                                    [r] => D11
                                                    [s] => 15
                                                    [t] => d
                                                )

                                            [v] => 2022-02-01
                                        )
```

Worksheet cell value is 2022 and so formated value as **2022**

```
            [3] => Array
                (
                    [type] => d
                    [name] => D11
                    [value] => 1905-07-14 00:00:00
                    [href] => 
                    [f] => 
                    [format] => mm-dd-yy
                    [r] => 11
                    [hidden] => 
                )
```

unixtimestamp function has a bug by this case.

**Comment by shuchkin** [comment]
send me xlsx file to sergey.shuchkin@gmail.com

**Comment by tgely** [comment]
Sent

**Comment by shuchkin** [comment]
fixed in 0.9.10

thx for bugreport

**Comment by tgely** [comment]
Thanks
